packages/app/src/components/inspector/ArrayController.tsx

```diff
@@ -30,10 +30,13 @@ interface ArrayControllerProps<Item> {
   title: ReactNode;
   sortable?: boolean;
   reversed?: boolean;
+  expanded?: boolean;
+  expandedContent?: ReactNode;
```

Comment:
For consistency with `renderItem` let's make this a "render prop" . I.e. `renderExpandedContent?: () => ReactNode`.

This also has the small perf benefit of not needing to run unless it's actually expanded and visible.